mysite/users/views.py

```diff
@@ -56,4 +56,16 @@ def profileUpdate(request, pk):
             edit.save()
             return redirect('profile')
     form = UserProfileForm()
-    return render(request,'user/profile.html',{'form': form})
\ No newline at end of file
+    return render(request,'user/profile.html',{'form': form})
+
+@login_required
+def editProfile(request):
+    if request.method == 'POST':
+        # server = get_object_or_404(Profile, pk=pk)
+        form = UserProfileUpdateForm(request.POST, instance=request.user)
+        if form.is_valid():
+            form.save()
+            messages.success(request,f'Your profile information updated successfully')
```

Comment:
If I am not wrong, there is a typo in the success message, please correct it.

Reply:
@Kajol-Kumari If you are talking about the "successfully" then it is correct.

Reply:
No, I am taking about the `,f'`

Reply:
Here ,f denotes formatted string. If we use any variables/expressions then it will be replaced by its value

Reply:
yeah, thanks for letting me know :)

Reply:
@Kajol-Kumari I have added the GIF. Please have a look